R/acf.R

```diff
@@ -63,7 +63,10 @@ ACF <- function(.data, ..., lag_max = NULL, demean = TRUE,
   compute_acf <- function(.data, value, ...){
     value <- enexpr(value)
     x <- eval_tidy(value, data = .data)
-    acf <- tail(as.numeric(acf(x, plot=FALSE, ...)$acf), -1)
+    acf <- as.numeric(acf(x, plot=FALSE, ...)$acf)
+    if(length(type) == 1 && type != "partial"){ # First indx already dropped
```

Comment:
The `match.arg` will choose the first of the vector within the default. If not specified, it will use `type[1]`.
This condition will never be triggered when `type` isn't specified (as `length(type) == 3)`).

Perhaps `type[1] != "partial"` would work better?

Reply:
Indeed. You're right I will push a correction. Was too fixated on the partial case.